packages/terra-site/src/examples/horizontal-divider/HorizontalDividerExample.jsx

```diff
@@ -0,0 +1,7 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import React from 'react';
+import HorizontalDivider from 'terra-horizontal-divider';
+
+const HorizontalDividerExample = () => <HorizontalDivider />;
```

Comment:
For one of our examples, could we show what it looks like when we have dividers between sections of text?

Reply:
Updated https://github.com/cerner/terra-core/pull/962/commits/0370a402365766a5bd10776bb052feff446f9c54#diff-073ae58a582f93cb086d6b8726511eddL13